src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -881,10 +881,9 @@ class DeclareStatementSegment(BaseSegment):
     """
 
     type = "declare_segment"
-    match_grammar = StartsWith("DECLARE")
-    parse_grammar = Sequence(
```

Comment:
Removing `parse_grammer` based on this comment: https://github.com/sqlfluff/sqlfluff/issues/3005#issuecomment-1088639544